sdk/cosmos/azure-cosmos-kafka-connect/src/docker/docker-compose.yml

```diff
@@ -0,0 +1,188 @@
+# Adapted from https://github.com/confluentinc/cp-all-in-one and https://github.com/simplesteph/kafka-stack-docker-compose
+version: '2.1'
+
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:6.0.0
```

Comment:
lets go over the versioning for these docker artifacts separately. Don't fully understand why / what for they are used etc. So, don't have an opinion what the right way to apply versioning is.

Reply:
it is used for spinning local confluent platforms.  I will change to use a higher version later, but this is a working version where we can run integration tests locally

Reply:
tracking item: https://github.com/Azure/azure-sdk-for-java/issues/38770